meteor-app/imports/ui/config/bike-assessment-schemas.js

```diff
@@ -186,22 +186,41 @@ export default [{
       title: "Enter Customer Details",
       properties: {
         b2bRefurbish: {
-          type: "boolean",
-          title: "Is this bike being refurbished by Back 2 Bikes?"
-        },
-        name: {
-          type: "string",
-          title: "Name"
-        },
-        phone: {
           type: "string",
-          title: "Phone Number"
-        },
-        email: {
-          type: "string",
-          title: "Email"
+          title: "Is this bike being refurbished by Back 2 Bikes?",
+          enum: ["Yes", "No", ""],
+          default: "No"
         },
       },
+      dependencies: {
+        b2bRefurbish: {
+          "oneOf": [
+            {
+              properties: {
+                b2bRefurbish: { enum: ["Yes", ""] }
```

Comment:
Please make this a boolean. Putting "Yes" into the database is very 1985, and you don't want that